model/src/main/kotlin/spdx/SpdxExpressionDefaultVisitor.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.spdx
+
+class SpdxExpressionDefaultVisitor : SpdxExpressionBaseVisitor<SpdxExpression>() {
+    override fun visitLicenseExpression(ctx: SpdxExpressionParser.LicenseExpressionContext): SpdxExpression {
+        return when (ctx.childCount) {
+            2 -> visit(ctx.getChild(0))
+            else -> throw Error("SpdxExpression has invalid amount of children: '${ctx.childCount}'")
+        }
+    }
+
+    override fun visitCompoundExpression(ctx: SpdxExpressionParser.CompoundExpressionContext): SpdxExpression {
```

Comment:
nit: you could import the 'CompoundExpressionContext' and also other innter classes like 'LicenseExpressionContext'
which gives a bit better readability. The names are unique enough to do that

Reply:
Done.